CONTRIBUTING.md

```diff
@@ -30,12 +25,11 @@ For more information, please see the Eclipse Committer Handbook:
 Eclipse m2e use mainly 2 channels for strategical and technical discussions
 
 * 🐞 View and report issues through uses GitHub Issues at https://github.com/eclipse-m2e/m2e-core/issues. _📜 Migration to GitHub tracker took place in March 2021, for older tickets, see https://bugs.eclipse.org/bugs/buglist.cgi?product=m2e 📜_
-* 📧 Join the m2e-dev@eclipse.org mailing-list to get in touch with other contributors aboantiqueut project organizationa and planning, and browse archive at 📜 [https://accounts.eclipse.org/mailing-list/m2e-dev](https://accounts.eclipse.org/mailing-list/m2e-dev)
```

Comment:
@mickaelistria do you still know what you meant by 'aboantiqueut'?

Reply:
I think it's just "about" with an unrelated pasted "antique" in the middle. "About" should be enough

Reply:
This makes sense. Thanks!